src/Service.Tests/Multidab-config.MsSql.json

```diff
@@ -15,7 +15,12 @@
     "graphql": {
       "enabled": true,
       "path": "/graphql",
-      "allow-introspection": true
+      "allow-introspection": true,
+      "nested-mutations": {
+        "create": {
+          "enabled": true
```

Comment:
Considering these files serve as a reference to the users - giving them an idea of how the resulting config will look like in the latest version, adding either `true` or `false` should be okay.

For integration tests, the config file that gets generated using the commands present in `*-commands.txt` gets utilized. But your question is valid and is one of the things I'm thinking through for adding tests in my other PR - essentially, how do we run integration tests with nested mutation operations as enabled/disabled. The challenge is that the config file gets generated once (and is utilized by all the integration tests) in the pipeline. 

One approach I'm considering - For tests specifically wanting to validate something when nested mutation is enabled/disabled, during the test setup phase, the config file can be read, and the `enabled` field can be edited to `true`/`false` as necessary and the file can be written back. Likewise, in the cleanup phase, the `enabled` field can be reset.

Reply:
Tests that rely on the value of `nestedmutations.enabled==true/false` should get their own custom config file created during test startup and not rely on generic test files used in generic mega classes like sqltestbase. Examples of this can be found in ConfigurationTests.cs

Reply:
Yeah, I agree.